react/features/app/actions.any.ts

```diff
@@ -120,7 +122,7 @@ export function maybeRedirectToTokenAuthUrl(
         const room = state['features/base/conference'].room;
         const { tenant } = parseURIString(locationURL.href) || {};
 
-        getTokenAuthUrl(config, room, tenant, true, locationURL)
+        getTokenAuthUrl((audioOnlyEnabled || startAudioOnly), config, room, tenant, true, locationURL)
```

Comment:
@saghul if reloadNow is dispatched and the participant joined low bandwidth mode or config startAudioOnly was set to true in the first place, this is expected, right? To rejoin on the same terms.